money-core/src/main/scala/com/comcast/money/core/japi/JMoney.java

```diff
@@ -0,0 +1,577 @@
+/*
+ * Copyright 2012 Comcast Cable Communications Management, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.comcast.money.core.japi;
+
+import com.comcast.money.api.Note;
+import com.comcast.money.core.Money;
+import com.comcast.money.core.Tracer;
+
+/**
+ * Java API for working directly with money from Java code
+ */
+public class JMoney {
+    private static Tracer tracer;
```

Comment:
this is never set (unless I missed it)?

Should always use `Money.Environment().tracer()`

Reply:
There is a `protected void setTracer` method, used by the unit tests to pass a mock `Tracer`